src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -85,6 +87,15 @@ public partial class KubernetesClientConfiguration
             return k8SConfiguration;
         }
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="KubernetesClientConfiguration"/> from pre-loaded config object.
+        /// </summary>
+        /// <param name="k8sConfig">A <see cref="K8SConfiguration"/>, for example loaded from <see cref="LoadKubeConfigAsync(string, bool)" /></param>
+        /// <param name="currentContext">Override the current context in config, set null if do not want to override</param>
+        /// <param name="masterUrl">Override the Kubernetes API server endpoint, set null if do not want to override</param>
+        public static KubernetesClientConfiguration BuildConfig(K8SConfiguration k8SConfig, string currentContext = null, string masterUrl = null)
```

Comment:
just naming to make this easier to understand.

```suggestion
        public static KubernetesClientConfiguration BuildConfigFromCurrent(K8SConfiguration k8SConfig, string currentContext = null, string masterUrl = null)
```

Reply:
Good plan.  I'm going to propose `BuildConfigFromConfigObject` to align with `...FromConfigFile` and because 'current' doesn't quite describe it.  But agree it needs additional qualification - thanks!